app/server/api/disbursement_api.py

```diff
@@ -0,0 +1,166 @@
+from decimal import Decimal
+
+from flask import Blueprint, request, make_response, jsonify, g
+from flask.views import MethodView
+from sqlalchemy import desc, asc
+from uuid import uuid4
+
+from server import db
+from server.schemas import credit_transfers_schema
+from server.models.user import User
+from server.models.disbursement import Disbursement
+from server.models.transfer_account import TransferAccount
+from server.models.credit_transfer import CreditTransfer
+from server.utils.auth import requires_auth
+from server.utils.transfer_filter import process_transfer_filters
+from server.utils.search import generate_search_query
+from server.utils.credit_transfer import make_payment_transfer
+from server.utils.transfer_enums import TransferSubTypeEnum, TransferModeEnum
+from server.models.utils import paginate_query, disbursement_credit_transfer_association_table
+from sqlalchemy.orm import lazyload, joinedload
+
+disbursement_blueprint = Blueprint('disbursement', __name__)
+
+class MakeDisbursementAPI(MethodView):
+
+    @requires_auth(allowed_roles={'ADMIN': 'admin'})
+    def post(self):
+        post_data = request.get_json()
+
+        # --- Handle Parameters ---
+
+        # HANDLE PARAM : search_stirng - Any search string. An empty string (or None) will just return everything!
+        search_string = post_data.get('search_string') or ''
+        # HANDLE PARAM : params - Standard filter object. Exact same as the ones Metrics uses!
+        encoded_filters = post_data.get('params')
+        filters = process_transfer_filters(encoded_filters)
+        # HANDLE PARAM : include_accounts - Explicitly include these users
+        include_accounts = post_data.get('include_accounts', [])
+        # HANDLE PARAM : include_accounts - Explicitly exclude these users
+        exclude_accounts = post_data.get('exclude_accounts', [])
+        disbursement_amount = abs(round(Decimal(post_data.get('disbursement_amount') or 0),6))
```

Comment:
i was like, "what's the round in there for?"
tried it out without the round... 
"ohhhh i seeeeee"

that being said, we should only expect whole integer 'cents' value no?